blockstack/explorer
# refactor: switch network modal to toast

## Description

This PR will hopefully fix several reported issues by removing the modal popup to switch networks. It is being replaced by a toast that simply lets the user know what network they are on. If the user is coming from a link which is on the opposite network, it will switch it automatically for them and show the toast.

For details refer to issue #481, #514, #518 (also check #360, #334, #349, #514)

![image](https://user-images.githubusercontent.com/6493321/130866073-193e5e24-b770-402b-b076-120a15c40047.png) 

## Type of Change

## Testing information

- Visit each page and switch networks to verify the toast is working as expected
- Open a transaction link from the wallet and see if switches network automatically
